generation/internal/php.go

```diff
@@ -34,6 +36,16 @@ func initialPhpSteps(ls labels.LabelSet) []config.Step {
 	return []config.Step{checkout, installPackages}
 }
 
+func phpImageVersion(composerVersion string) string {
+	version := "8.2.7"
+	if composerVersion != "" {
+		version = strings.TrimPrefix(composerVersion, "~")
+		version = strings.TrimPrefix(version, "^")
+		version = strings.TrimSuffix(version, ".*")
+	}
+	return "cimg/php:" + version + "-node"
```

Comment:
We don't have docker images for every patch version: https://hub.docker.com/r/cimg/php/tags?page=1&name=8.1.3

Should we instead look for the minor version (e.g `8.1`)?